pkgs/by-name/xd/xdg-ninja/package.nix

```
@@ -5,17 +5,18 @@
   makeWrapper,
   jq,
   glow,
+  nix-update-script,
 }:
 
 stdenvNoCC.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation {
```
Since we no longer need `version` for `rev`, we can remove the `rec`.